### ISSUE TITLE
item reloads each time i press a key

### ISSUE THREAD

**Issue by juliusknth** [PolymerLabs/todo-list#9]
I am getting this error when typing Uncaught TypeError: Cannot read property 'indexOf' of undefined.
Here is the link ( https://github.com/juliusknth/e-bima-fire ) to my github repo that I am facing the issue in. here is the live application hosted on firebase https://e-bima-fire.firebaseapp.com/#!/applications. The above app was derived from the todo repo

**Comment by juliusknth** [comment]
here is the live application https://e-bima-fire.firebaseapp.com/#!/applications

**Comment by evanomondi** [comment]
Am also experiencing this issue.

**Comment by juliusknth** [comment]
Hello Nicotu, i have download the latest build and still doesn't work.

**Comment by Nicotu** [comment]
Hi, @juliusknth. I wasn't having this issue before. I was having issue #3. I did a bower update, and then downloaded the latest firebase element release and replaced it in the bower_components folder. Firebase is working fine now, but I started having your same issue with the input field. I'm trying to dig a bit more, but so far, no clue.

**Comment by juliusknth** [comment]
thanks for the quick reply. cool, I'll also try figure it out from my side, cause I also faced  issue #3. and found the solution from comments in  issue #3.

**Comment by juliusknth** [comment]
Hi, @Nicotu Another issue I get is when I add a firebase-colletion (used to create a list of radio-buttons where a user can select on option) to an item in the list, when I type i now get this error Uncaught Error: unexpected key splices, How this can bring more light to the issue

**Comment by robdodson** [comment]
I just did a fresh checkout of the project and am not seeing this issue. It's difficult to know what's wrong in @juliusknth site because it's got a lot of changes from the original app.

@Nicotu @evanomondi Are either of you experiencing this issue with the original version of this app or have you made changes? If you're able to provide steps to reproduce I can try to debug it.

**Comment by juliusknth** [comment]
@robdodson, I just did --
--git clone git clone https://github.com/PolymerLabs/todo-list.git 
--npm install 
--bower update
--gulp serve
i am on windows 10
but I am still getting this error (Uncaught TypeError: Cannot read property 'indexOf' of undefined) each time i edit a todo from the todo list

**Comment by robdodson** [comment]
@juliusknth did you change your app to point at a different firebase instance?
https://github.com/PolymerLabs/todo-list/blob/master/app/index.html#L66

You need to set the location to your own firebase

**Comment by Nicotu** [comment]
@robdodson Unfortunately also in my case I've a different app. I've build this very simple app to try to isolate firebase problem I was dealing with before. So there's no login, but it's a simple page with the list of items and the input field. You may try to have a look at it https://luminous-fire-2392.firebaseapp.com .  To recreate the problem you can go to the "comments" section and try to edit an existing list item. The elements still gets updated in the database, but  every time I edit it, I get an error in my consolle.

I wasn't having this problem before, and no messages were appearing in my consolle. I din't change anything in my old code, but just ran a bower updated and replaced the firebase element with the last release (1.0.7). Let me know if I can do anything else to help

**Comment by robdodson** [comment]
Yeah it's hard to debug without some kind of reduced test case...
It seems like it may be related to upgrading to firebase-element 1.0.7 as I think that's the primary change I made.

@cdata have you ever seen errors like this?

**Comment by juliusknth** [comment]
I have just tried this with my own firebase instance but still facing the issue

**Comment by evanomondi** [comment]
I preety much have the same exact issue as @Nicotu

**Comment by robdodson** [comment]
@evanomondi without steps to reproduce I'm not able to fix the issue. Cloning the project from GitHub and running it seems to work fine. I need to know what has changed in your apps to zero in on the cause

**Comment by evanomondi** [comment]
Hi @robdodson I haven't changed anything I pulled the code from github. did npm install, bower install then gulp serve. I created a new toDo list and tried editing it  then got the error. you can find my code here: https://github.com/evanomondi/evanToDo

**Comment by robdodson** [comment]
@evanomondi I think you need to change the firebase instance in your code:
https://github.com/evanomondi/evanToDo/blob/master/app/index.html#L66

I think you need to create your own firebase instance and point it at that.
